src/neptune/metadata_containers/model.py

```diff
@@ -255,7 +258,15 @@ def get_url(self) -> str:
             sys_id=self._sys_id,
         )
 
-    def fetch_model_versions_table(self, *, columns: Optional[Iterable[str]] = None) -> Table:
+    def fetch_model_versions_table(
+        self,
+        *,
+        columns: Optional[Iterable[str]] = None,
+        limit: Optional[int] = None,
+        sort_by: str = "sys/creation_time",
+        ascending: bool = False,
+        progress_bar: Optional[Union[bool, Type[ProgressBarCallback]]] = None,
```

Comment:
Should `progress_bar` not be documented?

Reply:
We probably should document the `ProgressBarCallback` interface somehow.

Reply:
Right, just making sure since it was missing from the docstrings' Args list.

I'd offer to contribute directly, but I don't know how this ProgressBar is supposed to work.

Reply:
@AleksanderWWW Would you mind?

Reply:
:eyes:

Reply:
@normandy7 will try to propose docstrings for the `ProgressBarCallback` in this PR today

Reply:
Added an example using `click`, though I don't have a strong opinion if such a full example is necessary, but thought that it might make it easier to understand what the callback is about. Let me know what you think please

Reply:
Thanks, will do by EOD!

Reply:
The example was helpful, thank you. Maybe it's not "necessary" to include in the docstring in full, but I don't think it hurts.

I'll push some suggestions shortly; do let me know if you have any remarks.